cluster/kubemark/config-default.sh

```diff
@@ -24,7 +24,9 @@ source "${KUBE_ROOT}/cluster/gce/config-common.sh"
 GCLOUD=gcloud
 ZONE=${KUBE_GCE_ZONE:-us-central1-b}
 REGION=${ZONE%-*}
+NUM_NODES=${NUM_NODES:-$KUBEMARK_NUM_NODES}
```

Comment:
This won't work if KUBEMARK_NUM_NODES is not set. What you should do is:
```
KUBEMARK_NUM_NODES=${KUBEMARK_NUM_NODES:-100}
NUM_NODES=${NUM_NODES:-$KUBEMARK_NUM_NODES}
```
and remove the
```
NUM_NODES=${NUM_NODES:-100}
```
line